components/eamxx/src/share/field/field_impl.hpp

```diff
@@ -372,9 +372,14 @@ update (const Field& x, const ST alpha, const ST beta)
 
   // Determine if there is a FillValue that requires extra treatment.
   ST fill_val = constants::DefaultFillValue<ST>().value;
-  const auto& xtra_data = get_header().get_extra_data();
+  const auto& xtra_data = x.get_header().get_extra_data();
```

Comment:
I don't think this will compile anymore. We changed how we access the field extra data.